### ISSUE TITLE
Where in rutorrent change interface of curl request?

### ISSUE THREAD

**Issue by scriptzteam** [Novik/ruTorrent#1308]
Where in rutorrent change interface of curl request?

for example i wanna use eth0:12

**Comment by scriptzteam** [comment]
Probably found myself xD

https://github.com/Novik/ruTorrent/blob/254551cc08311cebebfa0e1843168f5b1163628c/php/Snoopy.class.inc#L81

https://github.com/Novik/ruTorrent/blob/254551cc08311cebebfa0e1843168f5b1163628c/php/Snoopy.class.inc#L494

Can we have setting for this??

**Comment by Nodens-** [comment]
I encountered the need to change that when I set up a configuration with advanced routing to tunnel ONLY torrent traffic through a vpn via binding rtorrent to the tunnel interface ip and making a iproute2 rule to route the traffic of the tun interface subnet to the vpn server.
 The moment I did that, check_port plugin stopped working as it was not checking for the forwarded port through the proper interface. Possibly other things break with this configuration.
 As I assume many people would try to setup what I did (route only torrent traffic through a VPN) there should be a configuration option for this. Or at least it should be set to the address bound in .rtorrent.rc. Later option provides less freedom but would work at least for this common case scenario.
